scripts/error_codes.py

```diff
@@ -205,6 +205,8 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
         "7053", # Unimplemented feature error (parsing stage), currently has no tests
         "2339", # SMTChecker, covered by CL tests
         "6240", # SMTChecker, covered by CL tests
+        "1733", # AsmAnalysis: expecting bool expression (everything is implicitly bool without types in Yul)
+        "9547", # AsmAnalysis: assigning incompatible types in Yul (whitelisted as there are currently no types)
```

Comment:
These two error codes still exist in terms of analysis checks, but these checks can not be tested for as they'll never be false. I think it may be good to keep that around in case we add back types.